gulpfile.js

```diff
@@ -54,15 +54,18 @@ gulp.task('all.min', ['sass'], () => {
   })
 })
 
-gulp.task('sass', ['sass-lint'], (cb) => {
-  gulp.src('src/sweetalert2.scss')
+gulp.task('sass', ['sass-lint'], () => {
+  return gulp.src('src/sweetalert2.scss')
     .pipe(sass())
     .pipe(autoprefix())
     .pipe(gulp.dest('dist'))
+})
+
+gulp.task('css.min', () => {
+  return gulp.src('dist/sweetalert2.css')
```

Comment:
We should declare `sass` as a dependency of this task, otherwise the `build` task runs this task and `sass` in parallel, and so this task will use a stale version of `dist/sweetalert2.css`.

Reply:
Done.